examples/text_embeddings_v2/export_v2.py

```diff
@@ -62,8 +62,8 @@ def parse_line(line):
   return token, values
 
 
-def load(file_path, parse_line_fn, num_lines_to_ignore=0,
-         num_lines_to_use=None):
+def load(file_path, parse_line_fn, num_lines_to_ignore,
```

Comment:
Why remove defaults? I think those were quite reasonable.

Reply:
The defaults are defined in the argument parse at the end for all parameters. If I keep the defaults in multiple places it might be more difficult to change later on. Now if we change the default in the argument parser only the change will be reflected all over the script.